bcdaworker/queueing/manager/que_test.go

```diff
@@ -165,3 +172,136 @@ func TestProcessJobFailedValidation(t *testing.T) {
 }
 
 // Test ALR startAlrjob
+func TestStartAlrJob(t *testing.T) {
+	// Set up data based on testfixtures
+	db := database.Connection
+	alrWorker := worker.NewAlrWorker(db)
+
+	// Create synthetic Data
+	// TODO: Replace this with Martin's testing strategy from #4239
+	// To do this, this requires bringing in worker que migrations
+
+	// Create the key value pair
+	exMap := make(map[string]string)
```

Comment:
Can we start using the strategy for the ALR data which lives in the bcda database?

I was playing around with the fixtures and did a data dump for the keyvalue field. Here's a snippet:
```
- created_at: 2021-04-08T10:03:51.843824-06:00
  updated_at: 2021-04-08T10:03:51.843824-06:00
  id: 54
  metakey: 1
  mbi: 4SN0A00AA00
  hic: lagP5i2Akdd9
  firstname: Sofia
  lastname: Miller
  sex: "1"
  dob: 1903-05-19T00:00:00Z
  dod: 0001-01-01T00:00:00Z
  keyvalue: 0x0eff81040102ff8200010c010c000004ff820000
```

The keyvalue encoded here is actually an empty map.